app/ports/templates/ports/portdetail.html

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
 {% endblock %}
 
-{% block title %}{{ req_port.name }}- Port |{% endblock %}
+{% block title %}Install {{ req_port.name }} Port on MacOS -  |{% endblock %}
```

Comment:
```suggestion
{% block title %}Install {{ req_port.name }} Port on macOS -  |{% endblock %}
```
Apple's official capitalization of "macOS" starts with a lower case "m". Other than that, this lg2m.